### ISSUE TITLE
12 word recovery works on desktop client but not android version

### ISSUE THREAD

**Issue by jaydouken** [spesmilo/electrum#2770]
Trying to transfer wallet from copay, generating a standard wallet with the 12 word recovery phrase only works on the desktop client and not android. The next button does not become usable after the 12th word is entered.

**Comment by SomberNight** [comment]
When and How was this 12 word seed generated? Was it generated using Copay?

**Comment by jaydouken** [comment]
I generated the wallet using copay, and I believe it also uses BIP44, which is apparently not supported as well (in the android app)

**Comment by ecdsa** [comment]
importing BIP39 seeds is not for newbies.
create a wallet with an electrum seed, and send your coins to that wallet.

**Comment by jaydouken** [comment]
That isn't an option for me, I've already done that twice trying to transfer wallets, and I'm not paying a $5+ transaction fee to transfer the small amount I have. There are currently over 30k unconfirmed transactions, and I've been waiting on a different wallet transfer for the past 3 days. If the android app refuses to accept my current wallet I will just use a different application.

**Comment by leafcutterant** [comment]
I too am in dire need of being able to import a BIP39 seed on Android.

Please guys, add this function to the mobile wallet.
